src/modules/auth/passport/oidc-strategy.ts

```diff
@@ -39,16 +46,47 @@ passport.use(
       );
```

Comment:
Since all the other error handling changed as well; this error is caught nowhere and will crash the application. Similarly to the rest is should use an `logger.error` and `callback(err)`.